projects/v2/src/assets/configuration.json

```diff
@@ -0,0 +1,366 @@
+{
+   
+"headerCount" : "11",
+
+"imgOptions" : ["PNG", "SVG", "Vega Spec", "Graph Data", "JSON-LD","OWL (RDF/XML)"],
+
+"masterSheetLink" :
+  "https://docs.google.com/spreadsheets/d/1tK916JyG5ZSXW_cXfsyZnzXfjyoN-8B2GXLbYD6_vF0/edit#gid=2034682742",
+
+"sheetId" : "1tK916JyG5ZSXW_cXfsyZnzXfjyoN-8B2GXLbYD6_vF0",
+
+"version1Url" : "https://hubmapconsortium.github.io/ccf-releases/v1.0/asct-b/",
+
+"playgroundSheetOptions" : [
+  {
+    "title": "Example",
+    "sheet": "example"
+  }
+],
+
+"sheetOptions" : [
```

Comment:
@vutukuriajay2241 I've never understood this. Why are we listing all the asct+b tables twice (here and in sheet-config.json)? This is not DRY. Am I missing something here?